src/GraphQL/Types/Schema.cs

```diff
@@ -63,6 +63,14 @@ public static ISchema For(string typeDefinitions, Action<SchemaBuilder> configur
 
         public bool Initialized => _lookup?.IsValueCreated == true;
 
+        // TODO: It would be worthwhile to think at all about how to redo the design so that such a situation does not arise.
+        private void CheckInitialized()
+        {
+            if (Initialized)
+                throw new InvalidOperationException(@"Schema is already initialized and sealed for modifications.
+There was an attempt to modify schema after it was initialized. You should call RegisterXXX methods only when Schema.Initialized = false.");
```

Comment:
I suggest we do not include line breaks within messages.

```suggestion
                throw new InvalidOperationException("Schema is already initialized and sealed for modifications. You should call RegisterXXX methods only when Schema.Initialized = false.");
```

Reply:
The message could be simplified even further, but whatever you think.

```suggestion
                throw new InvalidOperationException("Schema is already initialized and sealed for modifications.");
```

Reply:
If possible, I always write in the error message not only that something bad happened, but also how to prevent it. It's more friendly.